docs/data/data-grid/row-grouping/row-grouping.md

```diff
@@ -83,13 +85,13 @@ Use the `hideDescendantCount` property of the `groupingColDef` to hide the numbe
 ### Hide the grouped columns
 
 By default, the columns used to group the rows remains visible.
-For instance if you group by `"director"`, you will have two columns titled _Director_:
+For instance if you group by `"director"`, you have two columns titled _Director_:
```

Comment:
UI element labels should be in bold rather than italic.

```suggestion
For instance if you group by `"director"`, you have two columns titled **Director**:
```